src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -395,9 +395,16 @@ private PokemonTableModel(List<Pokemon> pokes, PokemonTable pt) {
 				type2Col.add(i.getValue(), StringUtils.capitalize(p.getMeta().getType2().toString().toLowerCase().replaceAll("none", "")));
 
 				PokemonMoveMeta pm1 = PokemonMoveMetaRegistry.getMeta(p.getMove1());
-				PokemonMoveMeta pm2 = PokemonMoveMetaRegistry.getMeta(p.getMove1());
+				PokemonMoveMeta pm2 = PokemonMoveMetaRegistry.getMeta(p.getMove2());
+				
 				Double dps1 = Double.valueOf(pm1.getPower())/Double.valueOf(pm1.getTime())*1000;
-				Double dps2 = Double.valueOf(pm2.getPower())/Double.valueOf(pm2.getTime()+1000)*1000;				
+				Double dps2 = Double.valueOf(pm2.getPower())/Double.valueOf(pm2.getTime()+500)*1000;
```

Comment:
Guess after the merge of my branch this line needs fixing/merging. I removed the unnecessary unboxing of those doubles.
So you have to do one other commit, @Ljaysoft .